converter/shieldhit/beam.py

```diff
@@ -60,16 +114,26 @@ class BeamConfig:
     beam_ext_y: float = 0.1  # [cm]
     sad_x: Optional[float] = None  # [cm]
     sad_y: Optional[float] = None  # [cm]
-    nstat: int = 10000
-    beampos: tuple[float, float, float] = (0, 0, 0)  # [cm]
-    beamdir: tuple[float, float, float] = (0, 0, 1)  # [cm]
+    n_stat: int = 10000
+    beam_pos: tuple[float, float, float] = (0, 0, 0)  # [cm]
+    beam_dir: tuple[float, float, float] = (0, 0, 1)  # [cm]
     delta_e: float = 0.03  # [a.u.]
     nuclear_reactions: bool = True
+    
+    modulator_source_filename: Optional[str] = None
+    modulator_source_file_content: Optional[str] = None
+    modulator_zone_id: Optional[int] = None
+    modulator_simulation: ModulatorSimulationMethod = ModulatorSimulationMethod.MODULUS
+    modulator_mode: ModulatorInterpretationMode = ModulatorInterpretationMode.MATERIAL
```

Comment:
Why isn't that optional as well ?

Reply:
those have default values based on default values from documentation